src/orion/benchmark/task/quadratics.py

```diff
@@ -0,0 +1,121 @@
+""" Defines the Quadratics task, as described in section 4.2 of the ABLR paper.
+
+[1] [Scalable HyperParameter Transfer Learning](
+    https://papers.nips.cc/paper/7917-scalable-hyperparameter-transfer-learning)
+"""
+from dataclasses import dataclass
+from logging import getLogger as get_logger
+from typing import Dict, Optional
+
+import numpy as np
+from orion.benchmark.task.base import BaseTask
+
+logger = get_logger(__name__)
+
+
+@dataclass
+class QuadraticsTaskHParams:
+    # NOTE: In the paper, section 4.2, the function is defined over R^3, but I'm
```

Comment:
I think this note should be part of the docstring of `QuadraticsTask` so that user can see it in sphinx's documentation.

Reply:
Fixed: moved to the docstring.